pylintrc

```diff
@@ -22,7 +22,7 @@ analyse-fallback-blocks=no
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
 extension-pkg-allow-list=numpy,
-                         scipy
+                         PySide2
```

Comment:
this tells pylint to load the compiled libraries for PySide2 so that is can see the symbols